config/rollbar.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+import Rollbar from 'rollbar'
+import { includes } from 'lodash'
+
+const rollbar = new Rollbar({
+  // https://rollbar.com/docs/notifier/rollbar.js/#configuration-reference
+  accessToken: process.env['ROLLBAR_ACCESS_TOKEN'],
```

Comment:
Have you already set this up?

Reply:
No, I'll let you do that and provide help to add the secret if you need it.